SparkPost/2web2ui
# FE-959 - Hibana style cleanup for accounts (subaccounts and users) pages

[FE-959](https://jira.int.messagesystems.com/browse/FE-959)

### What Changed
- Updates Subaccounts pages for Hibana to match mocks
- Updates Users pages for Hibana to match mocks
- Replaces hard coded max widths with newly-provided Matchbox design tokens in several spots in the app

### How To Test
- Verify the following routes look nice in both the OG theme and Hibana theme:
  + /account/subaccounts
  + /account/subaccounts/create
  + /account/subaccounts/:id
  + /account/users
  + /account/users/create
  + /account/users/edit/:id
- Verify these designs resemble the mocks
- Verify these features still function as intended

### To Do
- [x] Look in to failing integration tests?
- [x] Fix loading states in Hibana for subaccounts pages
